### PR TITLE
ssh: Add --command argument to pass along to ssh

### PR DESCRIPTION
**Pull request by Joelgranados**
A small optimization to make my workflow that much faster :)

### CODE REVIEW

File: cmd/ssh [hunk 1]
```diff
@@ -77,5 +82,6 @@ _ssh() {
   fi
 
   #shellcheck disable=SC2029
-  ssh "${args[@]}" localhost "$@"
+  echo "ssh ${args[@]} localhost $@"
```
Comment (birkelund) [inline cmd/ssh:85]: Think this debug line can be dropped?

**Comment by birkelund** [comment]
YES PLEASE! :)

File: cmd/ssh (hunk 1 above)
Comment (Joelgranados) [reply cmd/ssh:85]: oops. that can easily be ammended :)

Comment (Joelgranados) [reply cmd/ssh]: Done

**Pull request merged**